src/lib/components/dropdown-button/stories/index.stories.tsx

```diff
@@ -41,36 +41,189 @@ export const WithLabel: Story = {
+        button,
+        'It expects that the button cursor is pointer'
+      ).toHaveStyle('cursor: pointer');
+
+      expect(
+        button,
+        'It expects that the button has a display flex'
+      ).toHaveStyle('display: flex');
+
+      expect(
+        borderColor,
+        'It expects that the button dont have border-color'
+      ).toBe('rgba(0, 0, 0, 0)');
+    });
+
+    await step('Checking the button content', () => {
+      expect(
+        button.childElementCount,
+        'It expects that the button has 2 elements'
+      ).toBe(2);
+
+      expect(
+        (button.firstChild as HTMLElement).tagName,
+        'It expects that the button first child is a text'
+      ).toBe('P');
+
+      expect(
+        button,
+        `It expects that the button has the label ${args.label}`
+      ).toHaveTextContent('' + args.label);
+
+      expect(
+        svgElements,
+        'It expects that the button has one svg element'
+      ).toHaveLength(1);
+
+      expect(
+        textElements,
+        'It expects that the button has one text element'
+      ).toHaveLength(1);
+    });
+
+    clearAllMocks();
+
+    await step('Checking the button click event', async () => {
+      await userEvent.click(button);
+
+      expect(
+        args.onClick,
+        'It expects that when to dropdown is clicked, the onclic event is called'
+      ).toHaveBeenCalled();
+    });
+
+    await step('Checking the dropdown structure', async () => {
+      await userEvent.click(button);
+
+      const dropdown = canvas.getByLabelText('dropdown');
+      const computedStyle = getComputedStyle(dropdown);
+      const background = computedStyle.getPropertyValue('background');
+
+      expect(
+        background,
+        'It expects that the dropdown background is rgb(255, 255, 255)'
+      ).toContain('rgb(255, 255, 255)');
+
+      expect(
+        dropdown,
+        'It expects that the dropdown position is absolute'
+      ).toHaveStyle('position: absolute');
+
+      expect(dropdown, 'It expects that the dropdown top is 56px').toHaveStyle(
+        'top: 56px'
+      );
+
+      expect(
+        dropdown,
+        'It expects that the dropdown height is 114px'
+      ).toHaveStyle('height: 114px');
+
+      expect(
+        dropdown,
+        'It expects that the dropdown width is 241px'
+      ).toHaveStyle('width: 241px');
+
+      expect(
+        dropdown,
+        'It expects that the dropdown border is 1px solid'
+      ).toHaveStyle('border: 1px solid  rgb(198, 198, 202)');
+
+      expect(
+        dropdown,
+        'It expects that the dropdown border-color is rgb(198, 198, 202)'
+      ).toHaveStyle('border-color: rgb(198, 198, 202)');
+
+      expect(
+        dropdown,
+        'It expects that the dropdown padding is 8px'
+      ).toHaveStyle('border-radius: 8px');
+
+      expect(
+        dropdown,
+        'It expects that the dropdown shadow is rgba(13, 16, 23, 0.04) 0px 2px 4px -2px, rgba(13, 16, 23, 0.12) 0px 4px 8px -2px'
```

Comment:
you can simplify these titles so they don't get too big, you can just use the property and not the value